generated-code/api/plaidApi.js

```diff
@@ -0,0 +1,3053 @@
+"use strict";
```

Comment:
is all this generated code supposed to be checked into git? seems like it should just be a build artifact when publishing.

Reply:
I don't think tests will pass without it. Im open to other options though.